balances-adjustment/src/main/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationImpl.java

```diff
@@ -110,13 +128,17 @@ public CompletableFuture<BalanceComputationResult> run(Network network, String w
             }
         } while (context.getIterationNum() < parameters.getMaxNumberIterations() && result.getStatus() != BalanceComputationResult.Status.SUCCESS);
 
+        Reporter terminationReporter = reporter.createSubReporter("termination", "Termination");
```

Comment:
Termination?

Reply:
my bad, `Status` would probably be better

Reply:
Done